src/snowcli/app/api_impl/plugin/plugin_config_provider_impl.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+from snowcli.api import PluginConfigProvider
+from snowcli.api.plugin.plugin_config import PluginConfig
+from snowcli.config import cli_config
+
+
+class PluginConfigProviderImpl(PluginConfigProvider):
+    def get_enabled_plugin_names(self) -> List[str]:
+        config_path = ["snowcli", "plugins"]
```

Comment:
Maybe change it to class constant?

Reply:
Discussed and agreed that current version is ok.